uosoftwareclub/leetcode-scraper
# Profile scraper

We now have the ability to pull from LeetCode and update data in the MongoDB.

Testing:
1. `yarn dev` in core-service
2. run scraper.py (don't forget to install packages, just follow readme.md)
3. See the things get updated in console of core-service, logs of core-service and mongodb
